### PR TITLE
Introduces configuration interface for functional test container

### PR DESCRIPTION
**Pull request by erkanerol**
Signed-off-by: Erkan Erol <eerol@redhat.com>

We predict that we will need more and more configuration for our functional tests in the future. We decided to introduce a configuration interface that allows users to provide inputs for their environments.

In this PR, we are starting to use that interface for ConsoleQuickStart objects. 

Note that the new interface is implemented as backward compatible. You don't need to pass the flag if the current tests are OK for you. 

**Release note**:
```release-note
NONE
```

### CODE REVIEW

File: docs/functest-container.md [hunk 1]
```diff
@@ -27,23 +27,44 @@ kubectl run functest --image=quay.io/kubevirt/hyperconverged-cluster-functest:1.
...
+  quay.io/kubevirt/hyperconverged-cluster-functest:1.4.0-unstable \
+  --config-file /tmp/testconf    
+```
+
+The content of the configuration file must be a valid yaml representation of `TestConfig` in [config.go](../tests/func-tests/config.go)
+
+***Configuration File Example***
+```yaml
+quickStart:
+  testItems:
```
Comment (nunnatsa) [inline docs/functest-container.md:47]: Please provide the upstream example instead

File: docs/functest-container.md [hunk 2]
```diff
@@ -27,23 +27,44 @@ kubectl run functest --image=quay.io/kubevirt/hyperconverged-cluster-functest:1.
 - If your systems are running in a namespace different from `kubevirt-hypercongerged`, you have to set the environment variable `INSTALLED_NAMESPACE` 
 
 
-## Arguments
+## Arguments and Configuration
 
-The arguments passed to this container image are directly passed to `func-tests.test` binary in the image.
+The arguments passed to this container image are directly passed to `func-tests.test` binary in the image. See [the example](#example-command)
+
+There is a special flag in the test binary which allows you to provide test specific configuration. You can mount your configuration file into the image and pass it with the flag.
```
[the remark this answers is not in the document]
Comment (erkanerol) [reply docs/functest-container.md]: You are soooo right. Just updated.

**Comment by nunnatsa** [comment]
Wrong required lane name

/override Sanity Checks / Sanity Checks

**Comment by nunnatsa** [comment]
Trying again:

/override "Sanity Checks / Sanity Checks"

**Pull request merged**